packages/suite/src/components/suite/AppNavigationPanel/index.tsx

```diff
@@ -99,42 +21,27 @@ interface Props {
     maxWidth: 'small' | 'default';
     children?: React.ReactNode;
     navigation?: React.ReactNode;
-    navigationSticky?: React.ReactNode;
 }
 
+export const useNavHook = () => React.useContext(NavigationContext);
```

Comment:
What is it? (naming is weird)

Reply:
hook for accessing child props and not passing it through all the DOM tree

Reply:
Would be nice to rename `NavigationContext`, `NavigationProvider` and `useNavHook`. 
(Btw. I would not use hook for this).

And a comment would be nice too. Because the prop `inView` is not self-describing.

Reply:
bc4b072ebba461e68470f831178ea69760baf51c